pkg/cluster/serviceBinding.go

```diff
@@ -159,7 +159,7 @@ func performBinding(ctx context.Context, options *ServiceBindingOptions, ns stri
 		List(ctx, metav1.ListOptions{Limit: 1})
 	if err != nil {
 		if options.ForceUseOperator {
-			return errors.New(localizer.MustLocalize("cluster.serviceBinding.operatorMissing") + err.Error())
+			return err
```

Comment:
I just want to say I've removed `cluster.serviceBinding.operatorMissing`, because there is no such localization string.